modules/navier_stokes/src/functormaterials/INSFVEnthalpyFunctorMaterial.C

```diff
@@ -100,25 +106,32 @@ INSFVEnthalpyFunctorMaterial::INSFVEnthalpyFunctorMaterial(const InputParameters
 
     addFunctorProperty<ADReal>(NS::time_deriv(getParam<MooseFunctorName>(NS::specific_enthalpy)),
                                [this](const auto & r, const auto & t) { return _h->dot(r, t); });
+
+    // Note that we don't pick the fluid temperature name
+    addFunctorProperty<ADReal>(NS::T_fluid,
```

Comment:
Would be nice to have dicumentation that this object is creating this functor under the hood.